config/src/config/storage_config.rs

```diff
@@ -84,27 +84,27 @@ pub struct StorageConfig {
     pub enable_indexer: bool,
 }
 
-pub const NO_OP_STORAGE_PRUNER_CONFIG: StoragePrunerConfig = StoragePrunerConfig {
-    enable_state_store_pruner: false,
-    enable_ledger_pruner: false,
-    state_store_prune_window: 0,
-    ledger_prune_window: 0,
-    ledger_pruning_batch_size: 10_000,
-    state_store_pruning_batch_size: 10_000,
-    user_pruning_window_offset: 0,
+pub const NO_OP_STORAGE_PRUNER_CONFIG: PrunerConfig = PrunerConfig {
+    ledger_pruner_config: LedgerPrunerConfig {
+        enable_ledger_pruner: false,
+        ledger_prune_window: 0,
+        ledger_pruning_batch_size: 10_000,
```

Comment:
nit: change 10_000 to 0 since this is NO_OP config?

Reply:
Done